HostToController.h

```diff
@@ -53,6 +60,26 @@ class HostToController : public UnidirectionalProxy<HostToController> {
     }
 
     void transfer(uint8_t* buffer, uint32_t length) {
+        /* link layer expects a wsf message it will take ownership of
```

Comment:
We're working with Cordio link layer on a single target; other targets don't expect a wsf message and won't take ownership of it. Please fix that comment to explain the different path possible depending on the underlying driver.